lib/render/mayaToHydra/renderGlobals.h

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright 2019 Luma Pictures
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef __MTOH_RENDER_GLOBALS_H__
+#define __MTOH_RENDER_GLOBALS_H__
+
+#include <pxr/pxr.h>
+
+#include <pxr/base/gf/vec4f.h>
+#include <pxr/base/tf/token.h>
+#include <pxr/base/vt/value.h>
+
+#include <maya/MObject.h>
+
+#include "../../usd/hdMaya/delegates/params.h"
+
+#include <tuple>
```

Comment:
I don't see the use of tuple.

Reply:
Good catch - fixed in https://github.com/Autodesk/maya-usd/pull/231/commits/d1b349f8ff22401c38f9ae7986c78306b58e4144